apps/web/src/components/editor/index.tsx

```diff
@@ -152,7 +152,12 @@ export default function TabsView() {
                 ) : session.type === "conflicted" || session.type === "diff" ? (
                   <DiffViewer session={session} />
                 ) : (
-                  <MemoizedEditorView session={session} />
+                  <Flex sx={{ overflow: "hidden" }}>
+                    <MemoizedEditorView session={session} />
+                    {isTOCVisible && activeSessionId && (
+                      <TableOfContents sessionId={activeSessionId} />
+                    )}
+                  </Flex>
```

Comment:
Instead of wrapping this in its own Flex, we should move the TOC to its own Panel. That way it'll become automatically resizeable.